apps/site/src/views/Resources/sections/ApiResources/ApiResources.module.scss

```diff
@@ -0,0 +1,62 @@
+@use "bootstrap-utils" as utils;
+
+.row {
+	margin-bottom: 50px;
+}
+
+.column {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+}
+
+.card {
+	background-image: url("~@/assets/index_cards/api-description-card.png");
+	background-repeat: no-repeat;
+	background-position: center;
+	height: 343px;
+	width: 80vw;
```

Comment:
Issue: making the width a percentage limits the available space on narrower views. Using a Bootstrap Container as the parent would provide more flexible padding.

Reply:
Perhaps I am doing it wrong, but I couldn't see a difference between using a Bootstrap Container and how it is now.

Reply:
There should be a difference here, but we can discuss this more in a separate issue.